authentication_test.go

```diff
@@ -55,13 +35,29 @@ func TestAuthenticationWithCookiePassage(t *testing.T) {
 		require.NotNil(t, err)
 		assert.Empty(t, userID)
 	})
+
+	t.Run("valid auth token", func(t *testing.T) {
+		// need to create a new request to set a different cookie
+		req, err := http.NewRequest("GET", "https://example.com", nil)
+		require.Nil(t, err)
+		req.AddCookie(&http.Cookie{
+			Name:  "psg_auth_token",
+			Value: PassageAuthToken,
+		})
+
+		userID, err := psg.AuthenticateRequest(req)
+		require.Nil(t, err)
+		assert.Equal(t, "bEXIZKYyApgz5oWYc5WM9vfF", userID)
```

Comment:
If I'm understanding the env vars correctly, this could potentially be
```suggestion
		assert.Equal(t, PassageUserID, userID)
```

Reply:
So the userID in the JWT is not actually the userID stored in `PassageUserID`.  The other SDKs follow this same pattern for checking against the particular userID.